doc/classes/TreeItem.xml

```diff
@@ -36,6 +36,12 @@
 				Calls the [param method] on the actual TreeItem and its children recursively. Pass parameters as a comma separated list.
 			</description>
 		</method>
+		<method name="clear_buttons">
+			<return type="void" />
+			<description>
+				Remove all buttons from the [TreeItem].
```

Comment:
```suggestion
				Removes all buttons from all columns of this item.
```

Reply:
Exposing the method and the performance fix are separate commits for that reason!